modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -382,6 +382,25 @@ definition {
 				echo("The string '${fileContentItem}' exists on this file");
 			}
 		}
+
+		FileUtil.delete("${directoryPath}/${fileName}");
+	}
+
+	@summary = "Verify that the state manager is present on MDF Request detail page"
+	macro viewFilterMDF {
```

Comment:
Hi @juliosilveira77, this macro name should be 'filterMDF', since you are just filtering, and not viewing anything